lib/chrome-binary.js

```diff
@@ -0,0 +1,33 @@
+var exec = require('child_process').exec;
+var spawn = require('child_process').spawn;
+var Promise = require('bluebird');
+
+var ChromeBinary = function(data, options) {
```

Comment:
This looks like a twin brother/sister of the Executable class in chromium-phases.js.